### PR TITLE
Merge Debian/Ubuntu Dependencies

### PR DESCRIPTION
**Pull request by enigma9o7**
Debian 10 too old for QT5.  Ubuntu dependencies tested and work fine on Debian. Increase header size of distro title & moved Gentoo into alphabetical order

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,61 +302,45 @@ sudo pacman -S qt5-base \
     qtkeychain-qt5
 ```
 
-##### Gentoo Linux
-
+#### Debian 11+ / Ubuntu 22.04 
+<font size= “1”>*Older OS versions require a newer version of Qt5 than offered in official repositories.*</font>
```
Comment (deepbluev7) [inline README.md:306]: Is this font tag necessary?

**Comment by deepbluev7** [comment]
In general I like it, thanks! Only nitpick rn is the font tag, I guess, which seems superfluous?

File: README.md (hunk 1 above)
Comment (enigma9o7) [reply README.md:306]: No, sorry, I was trying to make it smaller, but it doesn't appear to do anything anyway.

**Comment by deepbluev7** [comment]
Thank you!

**Pull request merged**